lib/testing-components/TestUseStoryblok.jsx

```diff
@@ -0,0 +1,24 @@
+import {
```

Comment:
Do we need this test in this SDK?

Reply:
Hmm I bet we don't need it. Good catch! @alexjoverm what do you think?

Reply:
True!